testutil/network/network.go

```diff
@@ -162,6 +162,7 @@ type (
 	}
 )
 
+// New creates a new Network for mock for integration tests.
```

Comment:
```suggestion
// New creates a new Network for integration tests.
```

imo `mock` has a very specific meaning in tests, which is not applied here.

Reply:
We are creating a network mock, so how about "creates a new network mock for integration tests"?

Reply:
updated to the suggestion